plugins/node/opentelemetry-plugin-express/test/express.test.ts

```diff
@@ -90,8 +90,10 @@ describe('Express Plugin', () => {
       });
       const router = express.Router();
       app.use('/toto', router);
-      router.get('/:id', (req, res, next) => {
-        return res.status(200).end();
+      router.get('/:id', (req, res) => {
+        setTimeout(() => {
+          return res.status(200).end();
+        }, 5);
```

Comment:
out of curiosity, would be "setImmediate" enough here, or it needs to wait for something else ?

Reply:
Yeah that would be enough, do you prefer using setImmediate ? It's moslty to be sure we have done at least one eventloop tick.

Reply:
both works fine, was just curious why 5. But setImmediate would then explain it more straightforward I think

Reply:
EDIT: I've updated it to `setImmediate`, i would prefer for people not to think it's done to wait for something.